Customize Product view page help of many containers (#6990)
magento/devdocs: src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md

`catalog_product_view_type_simple.xml` | Layout from this file is applied to `simple` product only
`catalog_product_view_type_virtual.xml` | Layout from this file is applied to `virtual` product only

> ## Customize Product view page

Reviewer: ```suggestion
## Customize product view pages
```

> With the help gives numerous containers on the product page, You should simply reference the container and add blocks to it.

Reviewer: ```suggestion
Use containers on the product page to structure content in the layout. You can reference the container and add blocks to it.

Containers assign content structure to a page using container tags within a layout XML file. A container has no additional content except the content of included elements. Examples of containers include:
```

*  `product.info.main`
*  `product.info.price`
*  `product.info.stock.sku`
*  `product.info.form.content`
*  `product.info.extrahint`
*  `product.info.social`
*  `product.info.media`

> Sample of usage in the product page layout containers:

Reviewer: ```suggestion
### Example
```

> 

Reviewer: Please describe what the example does.

Reviewer: In this case, the sample code seems to be simple and intuitive, so it doesn't need the explanation of the result. Be mindful of future contributions however. More complex code samples usually require this type of explanation.
